From 42bcfa6360ea8b7e69533f9b78ad057fa7827a8c Mon Sep 17 00:00:00 2001
From: Anders Goncalves da Silva <andersgs@gmail.com>
Date: Sun, 28 Jul 2019 15:37:56 -0700
Subject: [PATCH] Update Makefile to play nice with bioconda

Bioconda now setups up a number of environment variables to correctly compile code on different platforms. To play nicely, and compile correctly, the Makefile should make use the LDFLAGS environmental var.
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 0a60ccc..55e93d5 100644
--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,7 @@ BINDIR=/usr/local/bin
 all:seqtk
 
 seqtk:seqtk.c khash.h kseq.h
-		$(CC) $(CFLAGS) seqtk.c -o $@ -lz -lm
+		$(CC) $(CFLAGS) seqtk.c -o $@ $(LDFLAGS) -lz -lm
 
 install:all
 		install seqtk $(BINDIR)
-- 
2.20.1 (Apple Git-117)

